content_scripts/vimium_frontend.coffee

```diff
@@ -94,7 +94,12 @@ settings =
 #
 # Give this frame a unique id.
 #
-frameId = Math.floor(Math.random()*999999999)
+window.frameId = Math.floor(Math.random()*999999999)
+
+# For debugging only. This logs to the console on the background page.
+window.bgLog = (args...) ->
```

Comment:
It isn't used.

I think we should leave it in though.  You had something similar in a PR some time ago.  It can be useful for debugging to have all messages in a single console.